src/main/java/uk/gov/register/filters/ContentSecurityPolicyFilter.java

```diff
@@ -12,6 +12,6 @@
 public class ContentSecurityPolicyFilter implements ContainerResponseFilter {
     @Override
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
-        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com");
+        responseContext.getHeaders().add(HttpHeaders.CONTENT_SECURITY_POLICY, "default-src 'self' www.google-analytics.com; font-src data:;");
```

Comment:
There is some work being done to move the fonts to a CDN so when this is complete then we could specify the source